docs-src/docs/reference/cli.md

```diff
@@ -405,6 +406,7 @@ l0 loadbalancer create [--port port ... | --certificate nameOrARN |
 * `--healthcheck-timeout timeout` - The length of time before the check times out (default: `5`).
 * `--healthcheck-healthy-threshold healthyThreshold` - The number of checks before the instance is declared healthy (default: `2`).
 * `--healthcheck-unhealthy-threshold unhealthyThreshold` - The number of checks before the instance is declared unhealthy (default: `2`).
+* `--idle-timeout idleTimeout` - The ELB Idle Timeout in seconds.
```

Comment:
I'm a little confused... is `--idle-timeout` a required or optional param?

Reply:
Optional, see here: https://github.com/quintilesims/layer0/pull/557/files/10b08bfdc37b962ea6383fbc772573a9b0fff523#diff-743886dffc7c416b92da823ddca0a301R395

Reply:
ok, you might wanna move this out of 'required parameters' then

Reply:
I think the truncated view of the code makes it look like it's under Required Parameters, but under the fold it should show that it is under the Optional Arguments section.

Reply:
ahhh opps, apologies then!